btp-core-codecs/src/main/java/org/interledger/btp/asn/codecs/AsnBtpErrorDataCodec.java

```diff
@@ -43,7 +44,7 @@ public AsnBtpErrorDataCodec(long requestId) {
         requestId,
         new AsnUtf8StringCodec(new AsnSizeConstraint(3)), //Code
         new AsnUtf8StringCodec(AsnSizeConstraint.UNCONSTRAINED), //Name
-        new AsnIA5StringCodec(AsnSizeConstraint.UNCONSTRAINED), //TODO Parse time
+        new AsnIA5StringCodec(AsnSizeConstraint.UNCONSTRAINED), //TriggeredAt
```

Comment:
BTP isn't using the 'new skool' fixed length sizes ?
https://github.com/interledger/rfcs/issues/440#issuecomment-405197100

Reply:
Guess not: https://github.com/interledger/rfcs/blob/master/asn1/BilateralTransferProtocol.asn#L72

Reply:
There is a lot of stuff in there that is outdated. Many of the BTP message types are never used, including `Prepare`.

https://github.com/interledger/rfcs/issues/445#issuecomment-409651286